dist/recline.js

```diff
@@ -3227,13 +3240,7 @@ my.SlickGrid = Backbone.View.extend({
     }
 
     function sanitizeFieldName(name) {
-      var sanitized;
-      try{
-        sanitized = $(name).text();
-      } catch(e){
-        sanitized = '';
-      }
-      return (name !== sanitized && sanitized !== '') ? sanitized : name;
+      return $('<div>').text(name).html();
```

Comment:
i think this is what come out of `./make cat`. Maybe something else that didn't got builded to `dist/recline.js` ?

Double checking right now